views/js/runner/plugins/tools/areaMasking/areaMasking.js

```diff
@@ -97,6 +97,7 @@ define([
                 .on('unloaditem', function (){
                     //remove all masks
                     _.invoke(self.masks, 'destroy');
+                    self.masks = [];
```

Comment:
The reference is removed from the array inside the destroy handler : https://github.com/oat-sa/extension-tao-testqti/pull/651/commits/38bcf43de60a1542e85ae61b94773d258b14ccf8#diff-baf76f6a005822fdf326f8434dffb6d1R78